### ISSUE TITLE
Modifying/Adding new CONCEPTS in the Wordlist pane is not reflected in the Edit/Align Cognates pane

### ISSUE THREAD

**Issue by maokoman** [digling/edictor#161]
Modifying or adding a new CONCEPT in the Edit Wordlist (EW) pane is not tracked in the Edit/Align Cognate Sets (EACS) pane. 

It is my understanding that the EW pane takes its reference from the specified columns (i.e. CONCEPT) whereas the EACS pane seems to index from the #@sorted_concepts list found at the end of the .csv file.

Modifying a CONCEPT or adding a new CONCEPT (or entire entry with a new concept name) in the EW pane does not get written to the #@sorted_concepts list index that the EACS pane employs. 
This is seen:

1. Within a session, a modified/new Concept will not immediately appear in the EACS pane (even though modifying alignments are immediately seen in the EW pane).
2. Saving the file with the modified/new concepts and opening it again. The modified or new concepts do not appear in the EACS pane.

As such, modified/new concepts cannot be aligned or grouped.  I have confirmed this problem with my current database as well as the east-ploynesian.tsv file from the Lingpy tutorial. The results are identical.

Further, I have made many changes to concepts in my current database. Is there anyway (in Edictor or LingPy) to resolve this issue? This would involve running through the CONCEPT column and (re)writing only those values to the #@sorted_concepts list.

This is also seen with regard to the filename. If working with a custom Edictor URL, the filename created with that URL does not update when using new versions of the database. Thus, If one creates a custom URL with a filename that contains a date (e.g. database20160625.csv), that filename will always be the filename that is saved. Even if one renames the filename on the following day and opens the new file (e.g. database20160626.csv) with the custom URL, the #@filename= does not update with respect to the updated filename used. It would be useful if this could be overwritten with whatever filename is being used.

**Comment by LinguList** [comment]
Hi, unfortunately, if only you add a completely new concept, EDICTOR needs to be reloaded, and you should clear all lines preceded by '#' at the moment. The reason is that it takes time to load the internal structure of the wordlist into different dimensions. So it was deliberately done in such a way that you should "reserve" enough concepts before starting to analyze.

We may consider changing this by adding some "reload" possibility, but it will always result in some unpredictable behavior if no complete reloading of the file is been done...

**Comment by LinguList** [comment]
> Further, I have made many changes to concepts in my current database. Is there anyway (in Edictor or LingPy) to resolve this issue? This would involve running through the CONCEPT column and (re)writing only those values to the #@sorted_concepts list.

Yes, the workaround is: delete all lines preceded by ```#``` and start afresh. It's much less effort spending the few minutes to reload and set it up than programming this from our side.

**Comment by LinguList** [comment]
The filename is a separate issue that probably should be solved. I just addressed this in #162

**Comment by LinguList** [comment]
since the first problem requires heavy work-around computationally, but only re-loading by the user, I'm closing this. The third problem is addressed in another issue.
